### PR TITLE
ibm_db2_cheat_sheet

### PR DESCRIPTION
**Pull request by mohan08p**
IA page : https://duck.co/ia/view/ibm_db2_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/db2.json [hunk 1]
```diff
@@ -0,0 +1,254 @@
+{
+    "id": "ibm-db2_cheat_sheet",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/db2.json]: ID should be `ibm_db2_cheat_sheet`, also file name should be `ibm-db2.json`.

File: share/goodie/cheat_sheets/json/db2.json [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    "template_type": "terminal",
+    "description": "A database software offers industry leading performance, scale, and reliability on your choice of platform from Linux to z/OS",
+    "metadata": {
+        "sourceName": "DB2 Commands",
+        "sourceUrl": "http://dublintech.blogspot.in/2011/10/db2-cheat-sheet.html"
+    },
+    "aliases": [
+        "ibm db2",
+	    "ibmdb2 cmd",
+	    "ibmdb2 commands",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/db2.json]: Remove `ibmdb2 cmd` and `ibmdb2 commands`.

File: share/goodie/cheat_sheets/json/db2.json [hunk 3]
```diff
@@ -0,0 +1,254 @@
+{
+    "id": "ibm-db2_cheat_sheet",
+    "name": "IBMdb2",
+    "template_type": "terminal",
+    "description": "A database software offers industry leading performance, scale, and reliability on your choice of platform from Linux to z/OS",
+    "metadata": {
+        "sourceName": "DB2 Commands",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/db2.json]: Use `Dublin Tech`.

File: share/goodie/cheat_sheets/json/db2.json [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+            },
+            {
+                "key": "DB2CMD",
+                "val": "Opens a command line processor"
+            },
+            {
+                "key": "DB2CC",
+                "val": "Opens db2 control center"
+            },
+	    {
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/db2.json]: Fix the indenting here.

File: share/goodie/cheat_sheets/json/db2.json [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+                "key": "DB2 GET AUTHORIZATIONS",
+                "val": "Get authorisation level"
+            }
+        ],
+        "Database commands via Command Line Processor (CLP)": [
+            {
+                "key": "DB2 GET DATABASE CONFIGURATION",
+                "val": "Gets current database configuration"
+            },
+	        {
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/db2.json]: More indentation needs to be fixed. Please go through the file and correct this.

File: share/goodie/cheat_sheets/json/db2.json [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+        "sourceUrl": "http://dublintech.blogspot.in/2011/10/db2-cheat-sheet.html"
+    },
+    "aliases": [
+        "ibm db2",
+        "ibm db2 shell"
+    ],
+    "section_order": [
+        "DB2 System Commands",
+        "Command Line Processor",
+        "Database commands via Command Line Processor (CLP)",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/db2.json]: @mohan08p lets capitalize c in commands too

**Comment by moollaza** [comment]
@mohan08p Thanks! This LGTM :+1:

**Comment by mohan08p** [comment]
thanks @moollaza :+1:

**Pull request merged**